source/components/atoms/Checkbox/Checkbox.stories.js

```diff
@@ -98,7 +98,7 @@ const Checkboxes = injectProps => {
           disabled="true"
```

Comment:
Invalid prop type for disabled, expected boolean.

Reply:
fixex